dvc/stage/utils.py

```diff
@@ -62,6 +62,7 @@ def fill_stage_outputs(stage, **kwargs):
         "plots_persist_no_cache",
         "outs_no_cache",
         "outs",
+        "outs_no_push",
```

Comment:
Let's not add these please. These are also used in CLIs.

I'll suggest a different way to create a stage in tests below.

Reply:
👍